### PR TITLE
Small changes part 1

### PR DESCRIPTION
**Pull request by zcoursey22**

### CODE REVIEW

File: client/src/components/update.jsx [hunk 1]
```diff
@@ -46,10 +47,20 @@ const Update = ({ update }) => {
...
-        {helpers.cutOffDesc(description)}
+        {description}
+      </p>
+    );
+  } else if (!backersOnly && shortDescription.length !== description.length) {
+    desc = (
+      <p>
+        {shortDescription}
+        <span style={{ textDecoration: 'underline' }}>
+          Read more
```
Comment (amvaid) [inline client/src/components/update.jsx]: @zcoursey22 you can either use the <u></u> element here, or a styled-component. Inline css is not best practice.

**Review by amvaid — COMMENTED** [review]
@zcoursey22 are you using styled-components yet? If so is that the correct implementation? I'm asking because its different from the documentation I've seen. If your positive it can be done this way than I'll prove.

**Comment by zcoursey22** [comment]
@amvaid I'll go and change all inline css to styled components, good catch.

**Review by amvaid — APPROVED** [review]
Looks good.

**Pull request merged**